jme3-core/src/main/java/com/jme3/math/Transform.java

```diff
@@ -490,19 +495,23 @@ public boolean equals(Object obj) {
      * Returns a string representation of the transform, which is unaffected.
      * For example, the identity transform is represented by:
      * <pre>
-     * Transform[ 0.0, 0.0, 0.0]
-     * [ 0.0, 0.0, 0.0, 1.0]
-     * [ 1.0 , 1.0, 1.0]
+     * Transform[0.0, 0.0, 0.0]
+     * [0.0, 0.0, 0.0, 1.0]
+     * [1.0, 1.0, 1.0]
      * </pre>
      *
      * @return the string representation (not null, not empty)
      */
     @Override
     public String toString() {
-        return getClass().getSimpleName()
-                + "[ " + translation.x + ", " + translation.y + ", " + translation.z + "]\n"
-                + "[ " + rot.x + ", " + rot.y + ", " + rot.z + ", " + rot.w + "]\n"
-                + "[ " + scale.x + " , " + scale.y + ", " + scale.z + "]";
+        return String.format(Locale.US, "%s"
+                + "[%.4f, %.4f, %.4f]%n"
+                + "[%.4f, %.4f, %.4f, %.4f]%n"
+                + "[%.4f, %.4f, %.4f]",
+                getClass().getSimpleName(),
+                translation.x, translation.y, translation.z,
+                rot.x, rot.y, rot.z, rot.w,
+                scale.x, scale.y, scale.z);
```

Comment:
Why the fixed decimal formatting?

Reply:
For uniformity reasons, otherwise some float variables might have different lengths